go.mod

```diff
@@ -13,12 +13,16 @@ require (
 	github.com/hashicorp/hcl/v2 v2.6.0
 	github.com/hashicorp/logutils v1.0.0
 	github.com/hashicorp/terraform v0.12.26
+	github.com/hashicorp/terraform-plugin-sdk v1.14.0 // indirect
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/mattn/go-colorable v0.1.6
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/sourcegraph/go-lsp v0.0.0-20181119182933-0c7d621186c1
 	github.com/sourcegraph/jsonrpc2 v0.0.0-20190106185902-35a74f039c6a
 	github.com/spf13/afero v1.2.2
 	github.com/terraform-linters/tflint-plugin-sdk v0.1.2-0.20200615160547-c1d3caf80fe0
+	github.com/terraform-providers/terraform-provider-aws v2.65.0+incompatible // indirect
```

Comment:
@bendrucker I noticed this line is removed when run `go mod tidy` because the code required this dependency is ignored on the build phase. This seems a little inconvenient. Any thoughts?

Reply:
Apparently this is a special case of `+build ignore` and any other constraint name will work.

https://github.com/golang/go/issues/29598
https://github.com/golang/go/issues/38061#issuecomment-603917218

Reply:
Should be resolved by https://github.com/terraform-linters/tflint/pull/809

Reply:
Awesome!